hypertrace-trace-enricher-api/src/main/java/org/hypertrace/traceenricher/trace/util/ApiTraceGraph.java

```diff
@@ -203,8 +203,18 @@ public ApiTraceGraph build() {
                   exitBoundaryEvent, child);
               edgeBetweenApiNodes.ifPresent(apiNodeEventEdgeList::add);
             } else {
-              LOGGER.warn("Exit boundary event {} can only have entry boundary event as child {}",
-                  exitBoundaryEvent, child);
+              LOGGER.warn("Exit boundary event with eventId: {}, eventName: {}, serviceName: {}," +
+                      " can only have entry boundary event as child. Non-entry child:" +
+                      " childEventId: {}, childEventName: {}, childServiceName: {}." +
+                      " traceId for events: {}",
```

Comment:
```suggestion
                      " traceId: {}",
```

Reply:
So 2 events are involved here. That's why I added the for events.